assets/js/renderer.js

```diff
@@ -355,8 +355,51 @@ $(document).ready(function () {
     // Center and reset node view
     resetZoomButton.click(() => nodeView.centerNode());
 
+    // Clear any saved splitter-related div widths from local storage when the app closes.
+    const clearDivs = () => {
+        if ('localStorage' in window && window['localStorage'] !== null) {
```

Comment:
I'm not sure it's necessary to check if localStorage is available. 

Typically, this is done on the web, because certain browsers do not support localStorage, but electron uses chromium (basically chrome) which has supported localStorage since v4.